src/utils/getAddressWithoutNetworkPrefix.ts

```diff
@@ -0,0 +1,5 @@
+function getAddressWithoutNetworkPrefix(address = ''): string {
```

Comment:
You can use `extractSafeAddress()` for this.

Reply:
I think that `extractSafeAddress` is also specific for the Safe Addresses that are present in the Url.
```javascript
export const extractSafeAddress = (): string => extractPrefixedSafeAddress().safeAddress
```

Reply:
This was an oversight, my bad.